### DISCUSSION TITLE
Exclude Options in Filter

### DISCUSSION THREAD

**Discussion by sidney-eliot** [gotson/komga#1797] [Ideas]
Allowing exclusions in the filter menu would open up so many more door when filtering for stuff. It can also help make up for lacking metadata. For example, if one has not added any language metadata to English media, as the bulk of the library consists of that language and only adds language metadata for foreign languages, then with exclusions one could filter by English language media by excluding the foreign languages.

The feature could be as simple as clicking a checkbox again to turn it into an X that excludes that value from the results.

**Comment by gotson** [comment]
this is already implemented in the latest version

**Reply by sidney-eliot** [reply]
Great, I assume this was fairly recently added then, I will check it out!
